src/NuGet.Core/NuGet.Packaging/Signing/Signatures/Signature.cs

```diff
@@ -274,7 +274,9 @@ protected Signature(SignerInfo signerInfo, SignatureType type)
                 }
             }
 
-            return new SignatureVerificationSummary(Type, SignatureVerificationStatus.Illegal, flags, timestamp, expirationTime);
+            var status = VerificationUtility.GetSignatureVerificationStatus(flags);
+
+            return new SignatureVerificationSummary(Type, status, flags, timestamp, expirationTime);
```

Comment:
Should we update the summary to not return the status and have any caller that cares about the status to calculate it with the flags?